__tests__/app.test.js

```diff
@@ -167,6 +157,79 @@ describe("app", () => {
 
   });
 
+
+  test('Returns Error with 404 when the article id it does not exist', () => {
+    const article_id = 50;
+    return request(app)
+   .get(`/api/articles/${article_id}/commentS`)
+    .expect(404)
+    .then(({body})=>{
+    expect(body.msg).toBe("Article not found")
+
+      });
+    
+  });
+
+  test('Returns Error with 404 if the article id is invalid', () => {
+    
+    return request(app)
+   .get("/api/articles/article/commentS")
```

Comment:
You could also make the test more explicit by putting something like `.get("/api/articles/invalid_article_id/comment")` . No biggie - just helpful.